### PR TITLE
STRF-8666: Adds test coverage for contains helper

### PR DESCRIPTION
**Pull request by pedelman**
## What? Why?
Adds test coverage for contains helper. It looks like we were previously missing coverage for this helper. Added some additional edge case testing.

## How was it tested?
Unit tests added.

----

cc @bigcommerce/storefront-team

### CODE REVIEW

**Review by junedkazi — APPROVED** [review]

File: helpers/contains.js [hunk 1]
```diff
@@ -21,6 +21,28 @@ const factory = () => {
...
+        case 'string':
+            return container.includes(value);
+        case 'object':
+            if (container) {
+                if  (Array.isArray(container)) {
+                    return container.includes(value);
+                }
+
+                // Check if the supplied value matches a top level value in the container object
+                return Object.values(container).some(objValue => {
```
Comment (junedkazi) [inline helpers/contains.js:35]: I would have liked for us to keep this simple like `Object.values(object1).includes(value);` but I guess if lodash is supporting complex type this make is super difficult for us

File: helpers/contains.js [hunk 2]
```diff
@@ -10,7 +10,7 @@
 const factory = () => {
     return function(container, value) {
         const options = arguments[arguments.length - 1];
-        const contained = container.includes(value);
+        const contained = containerIncludesValue(container, value);
```
Comment (jairo-bc) [inline helpers/contains.js:13]: What is the general reason to support more data types?

Comment (pedelman) [reply helpers/contains.js:13]: I wanted to make sure to preserve as much backward compatibility as possible. Lodash supported more options so its not as simple of a refactor as I initially thought, we also didnt explicitly test this case, but our public helper docs point out object as a possible value.
